styles/globals.css

```diff
@@ -20,3 +20,28 @@ a {
 * {
   box-sizing: border-box;
 }
+
+/* styles for top under construction banner start here*/
+
+.banner {
+  background: white;
+}
+
+.banner>div {
+  padding: 1rem;
+}
+
+.stripe {
+  background: repeating-linear-gradient(-55deg, #222, rgb(34, 34, 34) 10px, rgb(229, 194, 0) 10px, rgb(229, 194, 0) 20px);
+}
+
+.text {
```

Comment:
Most, if not all this styles can also be represented using Tailwind. Please check.

Reply:
Now there is nothing in global css related to Banner, almost everything added using tailwind and little bit added inline css